ember-welcome-page/src/components/welcome-page.ts

```diff
@@ -0,0 +1,45 @@
+import { getOwner } from '@ember/application';
+/* @ts-expect-error: Cannot find module '@ember/version' or its corresponding type declarations. */
+import { VERSION } from '@ember/version';
+import Component from '@glimmer/component';
+import './welcome-page.css';
+
+function isLatestVersion(): boolean {
+  const stableRegex = /^\d+\.\d+\.\d+$/;
+
+  return !stableRegex.test(VERSION as string);
+}
+
+export default class WelcomePageComponent extends Component {
+  get rootURL(): string {
+    const config = getOwner(this).factoryFor('config:environment');
+
+    if (config) {
+      return (config.class as unknown as { rootURL: string }).rootURL;
+    }
+
+    return '/';
+  }
+
+  get emberVersion(): string {
```

Comment:
The template creates a data attribute called `data-ember-version`. I'm not sure if this data attribute is used in other projects.

To my knowledge, the data attribute is only used for this project's rendering tests and used incorrectly. What the rendering tests really want to assert is that the links to the Ember Guides will be correct. Such assertion can be done without the data attribute.

In short, we could delete the getter `emberVersion`, if we can verify that the data attribute `data-ember-version` is not used by external projects.

Reply:
@ijlee2 just did a quick code search using Ember Observer, I'm only seeing `data-ember-version` used in this addon: https://emberobserver.com/code-search?codeQuery=data-ember-version

And that matches what I see when searching Github orgs:
- https://github.com/search?q=org%3Aember-cli+data-ember-version&type=code
- https://github.com/search?q=org%3Aember-learn+data-ember-version&type=code

Making the change **might** affect internal code at organizations that we can't see, but this looks non-breaking for the visible Ember ecosystem. Were you thinking of doing a major version bump due to the introduction of Typescript? Seems like that'd cover us if so

Reply:
Nice, thank you for checking. I'll go ahead and remove the getter then.

I forgot to clarify, my bad. A major version release will be needed because I dropped support for Node 12 and Ember 3.24 in #388.